cloud/azure/management/recovery/mode/backupjobsstatus.pm

```diff
@@ -119,8 +119,7 @@ sub new {
                                     "resource-group:s"      => { name => 'resource_group' },
                                     "filter-name:s"         => { name => 'filter_name' },
                                     "filter-counters:s"     => { name => 'filter_counters' },
-                                    "warning-status:s"      => { name => 'warning_status', default => '' },
-                                    "critical-status:s"     => { name => 'critical_status', default => '%{status} eq "Failed"' },
+                                    "lookback:s"            => { name => 'lookback', default => '86400'}
```

Comment:
@lchrdn please don't set default value here please to avoid changing the actual behavior.